worlds/oot_soh/location_access/overworld/hyrule_field.py

```diff
@@ -5,8 +5,305 @@
+        (Locations.HF_EAST_TREE_5, lambda bundle: (can_bonk_trees(bundle))),
+        (Locations.HF_EAST_TREE_6, lambda bundle: (can_bonk_trees(bundle))),
+        (Locations.HF_SOUTHEAST_TREE_1, lambda bundle: (can_bonk_trees(bundle))),
+        (Locations.HF_SOUTHEAST_TREE_2, lambda bundle: (can_bonk_trees(bundle))),
+        (Locations.HF_SOUTHEAST_TREE_3, lambda bundle: (can_bonk_trees(bundle))),
+        (Locations.HF_SOUTHEAST_TREE_4, lambda bundle: (can_bonk_trees(bundle))),
+        (Locations.HF_SOUTHEAST_TREE_5, lambda bundle: (can_bonk_trees(bundle))),
+        (Locations.HF_SOUTHEAST_TREE_6, lambda bundle: (can_bonk_trees(bundle))),
+        (Locations.HF_SOUTHEAST_TREE_7, lambda bundle: (can_bonk_trees(bundle))),
+        (Locations.HF_SOUTHEAST_TREE_8, lambda bundle: (can_bonk_trees(bundle))),
+        (Locations.HF_SOUTHEAST_TREE_9, lambda bundle: (can_bonk_trees(bundle))),
+        (Locations.HF_SOUTHEAST_TREE_10, lambda bundle: (can_bonk_trees(bundle))),
+        (Locations.HF_SOUTHEAST_TREE_11, lambda bundle: (can_bonk_trees(bundle))),
+        (Locations.HF_SOUTHEAST_TREE_12, lambda bundle: (can_bonk_trees(bundle))),
+        (Locations.HF_SOUTHEAST_TREE_13, lambda bundle: (can_bonk_trees(bundle))),
+        (Locations.HF_SOUTHEAST_TREE_14, lambda bundle: (can_bonk_trees(bundle))),
+        (Locations.HF_SOUTHEAST_TREE_15, lambda bundle: (can_bonk_trees(bundle))),
+        (Locations.HF_SOUTHEAST_TREE_16, lambda bundle: (can_bonk_trees(bundle))),
+        (Locations.HF_SOUTHEAST_TREE_17, lambda bundle: (can_bonk_trees(bundle))),
+        (Locations.HF_SOUTHEAST_TREE_18, lambda bundle: (can_bonk_trees(bundle))),
+        (Locations.HF_SOUTHEAST_TREE_19, lambda bundle: (can_bonk_trees(bundle))),
+        (Locations.HF_CHILD_SOUTHEAST_TREE_1, lambda bundle: (is_child(bundle) and can_bonk_trees(bundle))),
+        (Locations.HF_CHILD_SOUTHEAST_TREE_2, lambda bundle: (is_child(bundle) and can_bonk_trees(bundle))),
+        (Locations.HF_CHILD_SOUTHEAST_TREE_3, lambda bundle: (is_child(bundle) and can_bonk_trees(bundle))),
+        (Locations.HF_CHILD_SOUTHEAST_TREE_4, lambda bundle: (is_child(bundle) and can_bonk_trees(bundle))),
+        (Locations.HF_CHILD_SOUTHEAST_TREE_5, lambda bundle: (is_child(bundle) and can_bonk_trees(bundle))),
+        (Locations.HF_CHILD_SOUTHEAST_TREE_6, lambda bundle: (is_child(bundle) and can_bonk_trees(bundle))),
+        (Locations.HF_TEKTITE_GROTTO_TREE, lambda bundle: (can_bonk_trees(bundle)))
+    ])
+    # Connections
+    connect_regions(Regions.HYRULE_FIELD, world, [
+        (Regions.LW_BRIDGE, lambda bundle: True),
+        (Regions.LAKE_HYLIA, lambda bundle: True),
+        (Regions.GERUDO_VALLEY, lambda bundle: True),
+        (Regions.MARKET_ENTRANCE, lambda bundle: True),
+        (Regions.KAKARIKO_VILLAGE, lambda bundle: True),
+        (Regions.ZR_FRONT, lambda bundle: True),
+        (Regions.LON_LON_RANCH, lambda bundle: True),
+        # Todo: Uses 'Here(RR_HYRULE_FIELD)'
```

Comment:
Feel free to remove the comments and ignore the Here instances. They're used to handle time of day in conjunction with entrance rando, which are both not applicable to us, and will have to be solved differently for us when does become revelant.